tests/test.py

```diff
@@ -1,9 +1,59 @@
+"""Testing Synchronous and Asynchronous Extraction"""
+
+import os
+import sys
 import unittest
 
+from any_parser import AnyParser
+
+sys.path.append(".")
+sys.path.append("..")
+sys.path.append("../..")
+
+
+class TestAnyParser(unittest.TestCase):
+    """Testing Any Parser"""
 
-class TestNode(unittest.TestCase):
     def setUp(self):
-        pass
+        # get secret API key
+        self.api_key = os.environ.get("API_KEY")
+        if not self.api_key:
+            raise ValueError("API_KEY environment variable is not set")
+        self.file_path = "./examples/sample_data/stoxx_index_guide_0003.pdf"
+        # create instance of AnyParser
+        self.ap = AnyParser(self.api_key)
+
+    def test_sync_extract(self):
+        """Synchronous Extraction"""
+
+        # check for specific content in the markdown
+        md_output, total_time = self.ap.extract(self.file_path)
+        self.assertFalse(md_output.startswith("Error:"), total_time)
+        self.assertTrue(
+            md_output.startswith("STOXX INDEX METHODOLOGY GUIDE"),
+            "Extraction failed",
+        )
```

Comment:
good starting point. considering that we know the group truth as the notebook, we should test the percentage of match of the model output and the target to about 90% accuracy at least. This test is too simple.